CONTRIBUTING.md

```diff
@@ -71,6 +71,10 @@ This plugin uses the `wp-env` package (an [official WP.org package](https://deve
 
   Docker installation depends on your OS. [Please follow their official instructions](https://docs.docker.com/get-docker/).
 
+- MySQL
+
+  To run integration tests, you will need a local MySQL installation.
```

Comment:
Should we also suggest a guide on how to install mysql?

[https://formulae.brew.sh/formula/mysql](https://formulae.brew.sh/formula/mysql)

Reply:
Good idea. I've added some additional text there.